### ISSUE TITLE
make new installer follow licence requirements

### ISSUE THREAD

**Issue by luclu** [ethereum/mist#1171]
Licence file used to be enclosed with the zipped platform binary.

**Comment by sant9ga** [comment]
Installation master with path-choose step would be great! (now browser and wallet are tried to install at the same folder replacing each other on windows)

**Comment by evertonfraga** [comment]
It happens on linux as well.
Thanks for pointing out Sant9ga

**Comment by molitar** [comment]
Yeah I can't even use the new version because it installs WHERE I DO NOT WANT IT TO!  Making wallet useless! Give us back the zip that we can just copy over our files where I want the MIST wallet to be!

**Comment by CaptainSkywave** [comment]
..I agree ;)

**Comment by MrBrighty** [comment]
Yeah, I'm kinda a noob at all your coding staff, inspite of it I've learnt how to change blockchain binary via cmd in previous versions of Mist. But new installer is totaly not user friendly. I can't choose a path for block already downloaded. I tried to use --datadir, no results. 
And now, I'm trying to place hotfix geth into mist 0.8.2 folder.

**Comment by sant9ga** [comment]
@MrBrighty geth replacement should work.
About chaindata path - you can use symbolic links alternatively, it'll definitely work!

**Comment by Tektonike** [comment]
Im beginer and loose first instalation with HD problems.
New inatalation, learning "How to resstore my firsts accounts and my money ! rsrsrsrsr

**Comment by MrBrighty** [comment]
@sant9ga , as I understood correctly, you know the way, how to install new wallet to be based on already downloaded chain (to choose the path for blocks). Could you please describe actions how to do it? 
I can't sync the chain during 3 days. Tried to change the geth in 0.8.2, but it's stuck on another block 2,287,782. 

P.S. Little bit confused, why my and sant9aga comments were deleted.

**Comment by luclu** [comment]
@sant9ga sorry I did, as it seemed resolved off-topic question.
The issue you are looking for is https://github.com/ethereum/mist/issues/1178 (wipe out thing)..
And https://github.com/ethereum/mist/issues/1169 (stalled syncing (not just at block 2283415))
Please stick to those threads as this is completely unrelated - again... ;)

**Comment by sant9ga** [comment]
@MrBrighty I wanted to PM but didn't find out how.
I guess new mist downloads chaindata in a different folder. 
1) You should find this new chaindata folder.
2) Rename it (just in case)
3) Make symbolic link there to point on the old-one chaindata folder. I've done it with command promt, but there are some apps for that [http://www.howtogeek.com/howto/16226/complete-guide-to-symbolic-links-symlinks-on-windows-or-linux/](url)

**Comment by MrBrighty** [comment]
@luclu thanks for the answer :) . From my point, conversation of sant9aga and me is about "how to change datadir of new installer" firstly. But if you think, that I'm missing again, okay, I won't be against the little cleaning from me :)

P.S. @sant9ga  Thanks for that, I'll try to follow.
